diff --git a/tensorflow/core/platform/posix/port.cc b/tensorflow/core/platform/posix/port.cc
index 1939cf72..96c96f50 100644
--- a/tensorflow/core/platform/posix/port.cc
+++ b/tensorflow/core/platform/posix/port.cc
@@ -104,7 +104,7 @@ void* AlignedMalloc(size_t size, int minimum_alignment) {
   const int required_alignment = sizeof(void*);
   if (minimum_alignment < required_alignment) return Malloc(size);
 #ifdef TENSORFLOW_USE_JEMALLOC
-  int err = jemalloc_posix_memalign(&ptr, minimum_alignment, size);
+  int err = posix_memalign(&ptr, minimum_alignment, size);
 #else
   int err = posix_memalign(&ptr, minimum_alignment, size);
 #endif
@@ -120,7 +120,7 @@ void AlignedFree(void* aligned_memory) { Free(aligned_memory); }
 
 void* Malloc(size_t size) {
 #ifdef TENSORFLOW_USE_JEMALLOC
-  return jemalloc_malloc(size);
+  return malloc(size);
 #else
   return malloc(size);
 #endif
@@ -128,7 +128,7 @@ void* Malloc(size_t size) {
 
 void* Realloc(void* ptr, size_t size) {
 #ifdef TENSORFLOW_USE_JEMALLOC
-  return jemalloc_realloc(ptr, size);
+  return realloc(ptr, size);
 #else
   return realloc(ptr, size);
 #endif
@@ -136,7 +136,7 @@ void* Realloc(void* ptr, size_t size) {
 
 void Free(void* ptr) {
 #ifdef TENSORFLOW_USE_JEMALLOC
-  jemalloc_free(ptr);
+  free(ptr);
 #else
   free(ptr);
 #endif
