diff --git a/tensorflow/cc/ops/standard_ops.h b/tensorflow/cc/ops/standard_ops.h
index 98f53010..f70e866f 100644
--- a/tensorflow/cc/ops/standard_ops.h
+++ b/tensorflow/cc/ops/standard_ops.h
@@ -35,6 +35,5 @@ limitations under the License.
 #include "tensorflow/cc/ops/state_ops.h"
 #include "tensorflow/cc/ops/string_ops.h"
 #include "tensorflow/cc/ops/training_ops.h"
-#include "tensorflow/cc/ops/user_ops.h"
 
 #endif  // TENSORFLOW_CC_OPS_STANDARD_OPS_H_
